tests/spread/smoketests/multi-base/all/charmcraft.yaml

```
@@ -0,0 +1,20 @@
+name: test-charm
+type: charm
+title: test
+summary: test
+description: |
+  A charm recipe that uses a multi-base platform syntax to define
+  architecture independent charms for 22.04 and 24.04.
+
+platforms:
+  jammy:
+    build-on: [ubuntu@22.04:amd64]
+    build-for: [ubuntu@22.04:all]
+  noble:
+    build-on: [ubuntu@24.04:amd64]
+    build-for: [ubuntu@24.04:all]
+
+parts:
+  my-charm:
+    plugin: charm
```

Since the charm plugin can't actually build architecture-independent charms anymore due to `ops` dependencies and for the sake of speed, let's do:

```suggestion
    plugin: dump
```